test/unit/vl-map-search.test.html

```diff
@@ -205,6 +322,207 @@
         assert.equal(element.getTranslation('select.no_more_options'), text4);
       });
     });
+
+    suite('Lambert-coördinaat', () => {
+      test('of - een vrije tekst, geeft "undefined"', () => {
+        expect(LambertCoordinaat.of('test')).to.be.undefined;
+      });
+
+      test('of - "null", geeft "undefined"', () => {
+        expect(LambertCoordinaat.of(null)).to.be.undefined;
+      });
+
+      test('of - "undefined", geeft "undefined"', () => {
+        expect(LambertCoordinaat.of(undefined)).to.be.undefined;
+      });
+
+      test('of - een lege string, geeft "undefined"', () => {
+        expect(LambertCoordinaat.of('')).to.be.undefined;
+      });
+
+      test('of - een geldige coördinaat, geeft een LambertCoordinaat', () => {
+        const lambertCoordinaat = LambertCoordinaat.of('123456.78, 345232.04');
+
+        expect(lambertCoordinaat).not.to.be.undefined;
+        expect(lambertCoordinaat).to.deep.equal({
+          _x: 123456.78,
+          _y: 345232.04,
+        });
```

Comment:
Ik zou hier de x en y getters gebruiken en afzonderlijk assert/expect hierop uitvoeren.

Reply:
denk da kik da net andersom had gezegd :-)

Reply:
maar ik bedoelde daar vooral mee dat er in de testen dikwijls gedupliceerd stond/staat 
- test x van het resultaat
- test y van het resultaat
- test toString van het resultaat

Reply:
dus in combinatie met @coemans hieronder die zegt toString apart testen ...

Reply:
Ik denk dat nu bij het toevoegen van een variabele aan lambert-coordinaat dat al deze testen gaan falen want de `deepEqual` gaat falen, dus eventueel test functie introduceren maar deep equal lijkt me hier geen goed patroon.